pieces/scenario/deferred-atlaskit-drag-and-drop/card.tsx

```diff
@@ -99,6 +115,19 @@ export const Card = memo(function Card({ item }: { item: Item }) {
       <Suspense>
         <LazyDropIndicator edge={closestEdge} gap={'var(--card-gap)'} />
       </Suspense>
+      <div css={controlStyles}>
+        <MenuButton label={`controls for card ${itemId}`}>
+          <MenuItem>Edit</MenuItem>
+          <MenuItem>Share</MenuItem>
+          <MenuItem>Move up</MenuItem>
+          <MenuItem>Move down</MenuItem>
+          {orderedColumnIds
```

Comment:
I am now thinking that maybe that using `orderedColumnIds` might not be ideal as it would mean that all cards in a column would need to re-render when a column is reordered

Reply:
I've added a comment about this, as discussed :)